apps/test/unit/applab/designElements/photoSelectTest.js

```diff
@@ -0,0 +1,36 @@
+import {expect} from '../../../util/deprecatedChai';
+import library from '@cdo/apps/applab/designElements/library';
+
+function getRect(e) {
+  return {
+    left: parseInt(e.style.left, 10),
+    top: parseInt(e.style.top, 10),
+    width: parseInt(e.style.width, 10),
+    height: parseInt(e.style.height, 10)
+  };
+}
+
+describe('Applab designElements/photoSelect component', function() {
+  let e;
+
+  beforeEach(() => {
+    e = library.createElement(
```

Comment:
Replace `e` with a more reader-friendly name. i.e. `photoSelectComponent` or something similar

Reply:
(and in other areas in this file)